renovate.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base"
```

Comment:
`config:base` has been replaced with `config:recommended` so I've updated this.
Actually this is not related to the target of this PR but it should has no side-effect.

Reply:
Per https://github.com/search?q=repo%3Arenovatebot%2Frenovate%20config%3Abase&type=code, confirmed.

Reply:
better one here https://github.com/renovatebot/renovate/commit/f9e3e80e0c9c6a972d847f8740de5016a2bf698a

Reply:
not sure why they are not sending auto update on that.   They should as its rather obscure, how did you even run into this?